redash/destinations/__init__.py

```diff
@@ -34,6 +34,10 @@ def enabled(cls):
     def configuration_schema(cls):
         return {}
 
+    @classmethod
```

Comment:
I made a mistake when designing these classes by using class methods for these attributes. How about we use a class property:

```python
class BaseQueryRunner(object):
    deprecated = False
```

and then in the deprecated classes:

```python
class Url(BaseQueryRunner):
    deprecated = True
```

Reply:
I feel the same. I went for a decorator in order to remain compatible with the existing `classmethod` style and still allow a succinct way to declare deprecations at the top of the object being deprecated. 

I can drop the decorator and stick with class properties, however we can choose to keep using decorators as they will always stay on top and feel "more meta". WDYT?

Reply:
![image](https://user-images.githubusercontent.com/71468/61612823-42467200-ac68-11e9-90ac-1f9d56cc9f5c.png)

How about we move to class properties, but also keep the decorator to make sure it's visible that this object is deprecated?

Reply:
yeah that's what I meant :) 5a9e13c